Library/Formula/ipsumdump.rb

```diff
@@ -0,0 +1,16 @@
+class Ipsumdump < Formula
+  homepage "http://www.read.seas.harvard.edu/~kohler/ipsumdump/"
+  url "http://www.read.seas.harvard.edu/~kohler/ipsumdump/ipsumdump-1.84.tar.gz"
+  sha1 "3dc3ff9d97a65be25866d624a663be91e27d8628"
+  head "https://github.com/kohler/ipsumdump.git"
+
+  def install
+    system "./configure", "--prefix#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/ipsumdump", "--help"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!